Core/clim-basic/clim-basic.asd

```diff
@@ -24,7 +24,9 @@
                  (:file "region-bounding-rectangles")
                  (:file "region-transformations")
                  (:file "region-predicates")
-                 (:file "region-composition")))
+                 (:file "region-composition")
+                 (:file "region-set-predicates")
+                 (:file "region-set-composition")))
```

Comment:
Maybe also extract the region classes and associated utilities (`map-over-bands` and friends) into `region-sets.lisp`? `regions.lisp` is almost 2000 lines and would benefit from it, I think.

Reply:
map-over-bands is already moved to regin-utilities.lisp (and regions has 774 lines)

Reply:
OK.